x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/reducer.ts

```diff
@@ -126,6 +126,7 @@ export const policyDetailsReducer: ImmutableReducer<PolicyDetailsState, AppActio
     return {
       ...initialPolicyDetailsState(),
       location: action.payload,
+      license: currentLicense,
```

Comment:
Ahh . Ok. I see now. I was a bit confused as to what the issue was, but I see it now. When NOT on the policy details page, we reset state, but preserve both location ++ license data.

Was this problem also seen when viewing/editing the policy from Fleet?

Reply:
@paul-tavares no, I tested that case